Fixed backup naming scheme, date field and a typo.

minhazulOO7 opened the pull request:
For backup/backupManager.py:

1. Now it will show backup name as, `backup-example.com-01-01-01-Jan-01-2020` instead of `backup-example-01-01-01-Sun-Jan-2020`.

For plogical/backupUtilities.py

1. Now it will show date field as, `Jan-01-2020` instead of `01-01-01-Sun-Jan-2020`.
2. Fixed a typo.

minhazulOO7 committed:
Update backupManager.py

Fixed backup naming scheme

minhazulOO7 committed:
Merge pull request #1 from minhazulOO7/minhazulOO7-patch-1

Fixed backup naming scheme

minhazulOO7 committed:
Fixed `backupSite` view of `Date` field

Now it will show `Jan-01-2020` instead of `01-01-01-Jan-01-2020`

minhazulOO7 committed:
Merge pull request #2 from minhazulOO7/minhazulOO7-patch-1

Fixed `backupSite` view of `Date` field

usmannasir commented:
Naming scheme are used at many places, is it compatible with other functions of CyberPanel?

minhazulOO7 commented:
You can check the [commit](https://github.com/usmannasir/cyberpanel/pull/109/commits/c110553feb07437f94204ca39fa16900cd6f5fc9). I am checking for any issues. Will knock you if I find any.

minhazulOO7 commented:
@usmannasir created a website, checked it, made a backup, deleted that website, restored that website. In my end all are OK.

usmannasir commented:
I see only 2 lines changed, what is the benefit of this naming scheme?

minhazulOO7 commented:
@usmannasir a very useful use case IMO:
Suppose domain user named **example** have 10 sites. Now that person backups all of his/ her sites. All of the backups are at `/home/domain/backup`. He/ she then copied and pasted every backup files on another storage/ server etc. Assuming he/ she copied and pasted every backup files on a single directory. Now how can he/ she differentiate which sites backup is which? Also `M-D-Y` format is **WAY TO WRONG!**.  If I turn up the number to 100 domains (E.g., Resellers) it will be a **NIGHTMARE!**. That's why I think this naming scheme is better. Yes your POV maybe different but I think this naming scheme is suitable.

**Example:**

**Previously:**
`backup-example-01-03-01-Sun-Jan-2020`
`backup-example-01-06-01-Sun-Jan-2020`
`backup-example-01-09-01-Sun-Jan-2020`
and so on...

**Currently:**
`backup-example.com-01-01-01-Jan-01-2020`
`backup-example.net-01-01-01-Jan-01-2020`
`backup-example.org-01-01-01-Jan-01-2020`
and so on...

minhazulOO7 commented:
Also there's a **7 char limit** in domain users. That will be a different topic.

minhazulOO7 commented:
Oh! Please see this [commit](https://github.com/usmannasir/cyberpanel/pull/109/commits/5140e5831cc8eb2837a595ecf35f1c7636cc6059).

minhazulOO7 committed:
Fixed typo.

Line 214, `logging.CyberCPLogFileWriter.statusWriter(status, 'Meta data us ready..')` to `logging.CyberCPLogFileWriter.statusWriter(status, 'Meta data is ready..')`

minhazulOO7 commented:
Also fixed a typo, check [commit](https://github.com/usmannasir/cyberpanel/pull/109/commits/c8d26e31ddfa87e1911a74b944edcae0cf0f0560).

w3servicesdotnet commented:
Yes. Giving more logical view.

spyngamerman commented:
This modification has been implemented into the commit below with a slight variation of the date.
https://github.com/usmannasir/cyberpanel/commit/25848342f51542a009fa58ce58dd1059b8c13bec

Thanks for your awesome idea and I attributed it to you. > https://github.com/usmannasir/cyberpanel/commit/25848342f51542a009fa58ce58dd1059b8c13bec#r35436846

We will close this now since this is now implemented.

Keep on submitting awesome ideas or feedback :)
